init.lua

```diff
@@ -36,6 +40,9 @@ I hope you enjoy your Neovim journey,
 - TJ
 
 P.S. You can delete this when you're done too. It's your config now :)
+
+TODO: Include PDE video link? ;)
```

Comment:
Love this! But we should be sure that simply copying in init.lua still works. Last we checked - it doesn't.